email_notification/helpers/email_translations.py

```diff
@@ -0,0 +1,320 @@
+# ruff: noqa: E501
+from config.utils.commons import LanguageType
+
+EMAIL_TRANSLATIONS: dict[str, dict[LanguageType, str]] = {
+    # APPLICATION_HANDLED
+    "subjectApplicationHandled": {
+        "fi": "Hakemuksesi on käsitelty",
+        "en": "subjectApplicationHandled EN",
+        "sv": "subjectApplicationHandled SV",
+    },
+    "applicationHandled": {
+        "fi": "Näet tiedon käsittelyn tuloksesta",
+        "en": "applicationHandled EN",
```

Comment:
I guess we need translations for these from someone. Any critical ones that would prevent merging?

Reply:
I'll check but I think translations only exist for the application received email. Not critical, though, since they are always sent in Finnish while the UI language is not yet stored for applications (nor is user's preferred language set) -- and there are no active application rounds anyhow until the next year (by which time they should be implemented).

Reply:
EN & SV translations for application emails added in separate comments; cf. table in ticket https://helsinkisolutionoffice.atlassian.net/browse/TILA-3219